doc/api/http2.md

```diff
@@ -3124,4 +3131,4 @@ following additional properties:
 [`tls.TLSSocket`]: tls.html#tls_class_tls_tlssocket
 [`tls.connect()`]: tls.html#tls_tls_connect_options_callback
 [`tls.createServer()`]: tls.html#tls_tls_createserver_options_secureconnectionlistener
-[error code]: #error_codes
+[error code]: #error_codes
```

Comment:
A nit: missing line break at the end of the file.

Reply:
Oops! Fixed! 🔧